crates/ruff/resources/test/fixtures/pycodestyle/E714.py

```diff
@@ -4,11 +4,11 @@
 #: E714
 if not X.B is Y:
     pass
-#: E714
+
+#: Okay
 if not X is Y is not Z:
```

Comment:
I feel like this one is arguably a false negative, what do you think?

Reply:
How would you rewrite it?

Reply:
Yeah, I guess you're right -- I have no idea :joy: